docs/source/release/v5.1.0/diffraction.rst

```diff
@@ -5,42 +5,53 @@ Diffraction Changes
+- Control of the output binning of `Polaris.create_total_scattering_pdf` is no longer controlled by
+  the parameter `output_binning`, instead `delta_r` should be used.
+- In `Polaris.create_total_scattering_pdf` the Q space workspace can be rebinned before calculating
+  the PDF by being given an input `delta_q`.
+- running `Polaris.create_total_scattering_pdf` with `debug=true` will preserve the
+  `self_scattering_correction` workspace.
+- The Lorch filter in Polaris.create_total_scattering_pdf is no longer enabled by default and must
+  be enabled with the `lorch_filter` parameter.
+- The :ref:`LoadWAND <algm-LoadWAND>` algorithm now adds `duration` log to the workspace.
+- The :ref:`PDFFourierTransform <algm-PDFFourierTransform>` algorithm has been updated to allow for
+  the algorithm to be run on a PDF to obtain a spectrum density function.
+- SampleDetails.set_materials now differentiates between sample density and crystal density for converting
+  between pdf types.
+- The :ref:`CalculatePlaczekSelfScattering <algm-CalculatePlaczekSelfScattering>` algorithm now accepts
+  the crystalographic density of the sample to correct for the powder density.
+- Square beam profile of 5mm x 5mm added to the PEARL_Definition_new_lowangle instrument definition file.
 
 Bugfixes
 ^^^^^^^^
-- The fourier filter on Polaris.create_total_scattering_pdf no longer produces a jagged mark at the cut off point.
+- The fourier filter on `Polaris.create_total_scattering_pdf` no longer produces a jagged mark at the filter limit value.
 
 Engineering Diffraction
 -----------------------
+
+.. figure:: ../../images/EnggDiff_5_1_0.png
+   :class: screenshot
+   :width: 700px
+   :align: right
+
 New features
 ^^^^^^^^^^^^
-- New algorithm for estimating background of powder spectra :ref:`EnggEstimateFocussedBackground <algm-EnggEstimateFocussedBackground>` using iterative smoothing.
-- Mantid fitting capability added to fitting tab of Engineering Diffraction UI (with simpler fitpropertybrowser providing only relevant options).
-- Loading focussed runs into the fitting tab now shows the proton charge weighted average and standard deviation of select log values (set in the settings) in a group of table workspaces.
-- Mantid fitting capability added to fitting tab of Engineering Diffraction UI.
+- :ref:`EnggEstimateFocussedBackground <algm-EnggEstimateFocussedBackground>` is a new algorithm for
+  estimating background of powder spectra using iterative smoothing.
+- The fitting tab of Engineering Diffraction UI now functions with standard Mantid fitting capability,
+  but with a simplified browser.
+- Loading focussed runs into the fitting tab now creates a group of table workspaces containing the
+  proton charge weighted average and standard deviation of select log values (set in the settings).
 - Added support for a Circular Beam Profile for use with Monte Carlo Absorption
```

Comment:
Possible I'm a bit late with this comment but just found some time to have a look through these release notes for diffraction.
This comment about the circular beam profile is in the wrong place. It should be in the framework.rst file next to the changes to the MonteCarloAbsorption algorithm. It doesn't relate to engineering diffraction

Reply:
I'll remove it from the notes